### ISSUE TITLE
How to find out the NuGet version?

### ISSUE THREAD

**Issue by philcarbone** [Microsoft/vsts-tasks#2050]
I am having an issue, posted here:
[https://github.com/AutoMapper/AutoMapper/issues/1499](https://github.com/AutoMapper/AutoMapper/issues/1499)

And I need to figure out which version of NuGet is running.

**Comment by philcarbone** [comment]
Posted to community board: https://social.msdn.microsoft.com/Forums/en-US/2a954c61-8b67-40c2-a28b-0784517555e2/when-running-the-nuget-pack-command-i-get-the-error-automapper-already-has-a-dependency-defined?forum=tfsbuild

**Comment by ericsciple** [comment]
If you call nuget.exe without any arguments, it prints the version number at the top of the usage info.

**Comment by ericsciple** [comment]
@jascam, @IvyMH  - fyi

**Comment by philcarbone** [comment]
Looks like the Hosted Build Server is running NuGet 3.3.0.212

**Comment by philcarbone** [comment]
I was able to fix/workaround the issue by putting a PowerShell script in to download the latest NuGet. Then I pointed all of the NuGet tasks to this new nuget.exe. Pros: builds are working again, Cons: each build downloads NuGet again, causing unnecessary load on NuGet,org. 

Here is my PowerShell:

$sourceNugetExe = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
$targetNugetExe = "$(build.sourcesdirectory)/nuget.exe"
Invoke-WebRequest $sourceNugetExe -OutFile $targetNugetExe
Set-Alias nuget $targetNugetExe -Scope Global -Verbose
nuget
